enterprise/frontend/src/embedding-sdk-bundle/types/dashboard.ts

```diff
@@ -41,6 +42,19 @@ export type DashboardEventHandlersProps = {
    * Callback that is called when the dashboard is loaded without cards.
    */
   onLoadWithoutCards?: (dashboard: MetabaseDashboard | null) => void;
+
+  /**
+   * A callback function that triggers when a question is opened from a dashboard card
+   * or when the user changes the visualization type of a question.
+   *
+   * @param display the new display type
+   */
+  onVisualizationChange?: (display: CardDisplayType) => void;
```

Comment:
tiny nit: just to use the same terminology.

```suggestion
   * @param visualization the new visualization type
   */
  onVisualizationChange?: (visualization: CardDisplayType) => void;
```